src/components/ConnectedList.jsx

```diff
@@ -1,15 +1,23 @@
 import React from 'react'
-import { translate } from 'cozy-ui/react/I18n'
 import { Link } from 'react-router'
+
+import Icon from 'cozy-ui/react/Icon'
+import { translate } from 'cozy-ui/react/I18n'
+
 import KonnectorList from './KonnectorList'
 
+import addAccount from '../assets/icons/icon-plus.svg'
+
 const ConnectedList = ({ t, connectors, children }) => (
   <div className='content'>
     <div className='con-top-bar'>
       <h1>{t('nav.connected')}</h1>
       <Link to='/category/all'>
         <button className='coz-btn coz-btn--regular'>
-          {t('add_account')}
+          <Icon
+            icon={addAccount}
+            className='con-icon--add'
```

Comment:
You should be able to use directly the props `height` and `width` instead of adding a css class only for this, shouldn't you?

Reply:
For an unknow reason, it doesn't work, the icon disappear when I add width/height

Reply:
Seems weird, but let's keep it like that it's correct too